02-Logging/02-Writing-to-a-File.ipynb

```diff
@@ -0,0 +1,509 @@
+{
```

Comment:
<p>I would suggest a minor restructuring here. Rather than immediately jumping to logging to GSD files, I would show a modification of the previous notebook where you specify an <code>output</code>  for <code>Logger.Table</code>  and show that you get a file. Then, move the <code>Binary file formats</code>  section from above down to here, and show how you can take advantage of that to increase the logging periodicity while keeping the file size small, and optionally also log vector quantities etc.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>